easybuild/easyconfigs/l/libpsml/libpsml-1.1.7-foss-2016b.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = 'ConfigureMake'
+
+name = 'libpsml'
+version = '1.1.7'
+
+homepage = 'https://launchpad.net/libpsml'
+description = """LibPSML provides a Fortran API to parse files in the
+ PSeudopotential Markup Language (PSML) format."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['https://launchpad.net/libpsml/trunk/1.1/+download/']
+
+checksums = ['3f3189d71ec1ce83e00e477ebbf6c516']
```

Comment:
@pouillon Please use a `SHA256` checksum (and drop the empty line above)

Reply:
Will only use SHA256 from now on.